Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -39,6 +39,18 @@ class AppStoreConnectService {
         return operation.execute(with: dependencies)
     }
 
+    func inviteBetaTesterToGroups(with options: InviteBetaTesterOptions) throws -> AnyPublisher<BetaTester, Error> {
+        let dependencies = InviteTesterOperation.Dependencies(
```

Comment:
@adamjcampbell I have to say that this dependencies stuff is just plain weird.  I get what is happening but it makes me go "wat?" when I see it.

Reply:
Yeah I know, I wanted plain function injection in order to be able to test the operations. 

Because all the requests are generic function calls made it quite interesting. There’s probably a better way to do it. 

I remember trying to make a protocol for a generic Requestor and running into issues.